intg/src/main/java/org/apache/atlas/model/typedef/AtlasBaseTypeDef.java

```diff
@@ -213,6 +214,14 @@ protected AtlasBaseTypeDef(AtlasBaseTypeDef other) {
         }
     }
 
+    public boolean isIndexFailed() {
+        return indexFailed;
```

Comment:
lets have the names in +ve sense and also rename the variable to express the intent. Make sure you have flags update accordingly
```
- isIndexFailed -> IsIndexCreated
- indexFailed -> indexCreated
```